database/seedPostgres.js

```diff
@@ -0,0 +1,73 @@
+const {StaySummaryTest} = require('./mockDatabase.js');
+const { summaries } = require('./postgresSchema.js');
+const { db } = require('./postgresDB.js');
+
+var randomIntLessThan = (input) => {
+  return Math.floor(Math.random() * input);
+}
+
+const PARTY_MODE = true;
+
+ const stayTypes = PARTY_MODE ?
+ ['Entire home', 'Private room', 'Treehouse', 'Entire bungalow', 'Entire camper', 'Studio apartment', 'Entire cabin', 'Private loft', 'Empty lot', 'Entire guest suite', 'Entire guesthouse', 'Entire condominium', 'Tiny house', 'Spy mansion', 'Haunted house', 'Bomb shelter', 'Bunkbed fort', 'Entire spacious trunk of car', 'Medieval castle', 'Entire Spice Bus', 'Airplane']
+ :
+ ['Entire home', 'Private room', 'Treehouse', 'Entire bungalow', 'Entire camper', 'Studio apartment', 'Entire cabin', 'Private loft', 'Empty lot', 'Entire guest suite', 'Entire guesthouse', 'Entire condominium', 'Tiny house'];
```

Comment:
Looks like `PARTY_MODE` is always true here, so wondering you you have this ternary operator to define `stayTypes`

Reply:
This was previously in the code base, but you are right that it can be taken out.